.github/workflows/ci.yaml

```
@@ -1093,7 +1093,7 @@ jobs:
           asset_content_type: text/plain
 
       - name: Publish Release
-        run: ./project/scripts/sbtPublish ";project scala3-bootstrapped ;publishSigned ;sonatypeBundleRelease"
+        run: ./project/scripts/sbtPublish ";project scala3-bootstrapped ;publishSigned ;sonatypeCentralUpload"
```

We are not publishing to the Sonatype Central repository. 

See:
- https://github.com/xerial/sbt-sonatype/blob/f9a8b84c26a1076c12cf45e3368677f56450aa7a/src/main/scala/xerial/sbt/Sonatype.scala#L225C5-L225C33
- https://github.com/xerial/sbt-sonatype/blob/f9a8b84c26a1076c12cf45e3368677f56450aa7a/src/main/scala/xerial/sbt/Sonatype.scala#L194

```suggestion
        run: ./project/scripts/sbtPublish ";project scala3-bootstrapped ;publishSigned ;sonatypeBundleUpload"
```